python/gilded_rose/updater/factory.py

```diff
@@ -0,0 +1,20 @@
+"""Factory for the updates."""
+from typing import Optional
+
+from functools import partial
+
+from gilded_rose import dtypes
+from gilded_rose.updater import constants, standard, utils
+
+
+def get_updater(item: dtypes.Item) -> Optional[utils.Updater]:
+    """Get the updater for the item."""
+    if item.name == constants.CONJURED_KEY:
```

Comment:
As they referenced '"Conjured" items' with an s on items here https://github.com/emilybache/GildedRose-Refactoring-Kata/blob/775d8758c2f35cd734f28e9fb4e773ecab1a9f26/GildedRoseRequirements.txt#L29. I understood it to mean; there can be different types of conjured items such as "Conjured health potion", "Conjured cheese burger" etc. So I interpretted it as dont test for equality, test for inclusion. But I could be wrong so dont change the code. Just go back for clarification etc